### DISCUSSION TITLE
Break compatibility with older typescript version, cut new major release

### DISCUSSION THREAD

**Discussion by danielpza** [LeDDGroup/typescript-transform-paths#278] [General]
Hey @nonara, I'm opening this issue to coordinate on when to cut a new release and remove compatibility with older typescript versions. Want to hear your thoughts.

I'm thinking with the new major version we can break support of typescript 3.x and 4.x, maybe even <5.5.x altogether (see https://github.com/LeDDGroup/typescript-transform-paths/issues/266)

**Some questions**

- any other breaking changes we want to include in the new major version?
- which is the minimum typescript version we want to support?

**Comment by danielpza** [comment]
I have started some work in the [next](https://github.com/LeDDGroup/typescript-transform-paths/tree/next) branch

**Reply by nonara** [reply]
Hey, Daniel! Good timing. I'm about to be on this as well. Long overdue — apologies! I've been slammed.

Can you send me an email to ron at nonara dot com with your slack info? We can chat and approach getting to 0 issues.
